src/reactviews/common/locConstants.ts

```diff
@@ -132,6 +133,8 @@ export class LocConstants {
                     args: [objectType],
                     comment: ["{0} is the object type"],
                 }),
+            retryUpdatingDatabase: l10n.t("Retry updating database"),
```

Comment:
The text for these buttons are a little long.  Maybe something simpler like "Retry" and "Back to preview".

@jdesai-24

Reply:
@Benjin, I agree, we can add "Retry" and "Back to preview". Do we need a close button, I think a cross icon would suffice. 
For the error message, can we break down the message into different points which will be easier to read?